api/src/main/resources/workbench-api.yaml

```diff
@@ -350,6 +350,37 @@ paths:
           description: Bad request
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/link-ras-account":
+    post:
+      tags:
+      - profile
+      consumes:
+      - application/json
+      description: Link user with a RAS account
+      operationId: linkRasAccount
+      parameters:
+      - in: body
+        name: authcode
+        description: The authorization code from RAS.
+        schema:
+          "$ref": "#/definitions/RasLinkRequestBody"
+      responses:
+        200:
```

Comment:
We will probably want something like 403 or 412 to handle the case where the user presents a valid credential, but the account does not meet the access requirements (e.g. doesn't have eRA commons in the passport)

Reply:
That make sense. added 403.